x-pack/legacy/plugins/siem/server/utils/build_query/create_options.ts

```diff
@@ -47,3 +60,23 @@ export const createOptions = (
       .map(field => field.replace(fieldReplacement, '')),
   };
 };
+
+export const createOptionsPaginated = (
```

Comment:
I do not think that we need it, `createOptions` is doing exactly the same thing

Reply:
<img width="1119" alt="Screen Shot 2019-07-10 at 8 04 58 AM" src="https://user-images.githubusercontent.com/6935300/60975588-737a8600-a2e9-11e9-8d38-6b7725ec4a72.png">
They take different arguments ^^